src/main/java/seedu/duke/ui/AppUi.java

```diff
@@ -113,6 +114,17 @@ public void printHelpMenu() {
         helpCommand.execute(financialList);
     }
 
+    /**
+     * Executes the exit command and returns the status of the application.
+     * 
+     * @return {@code true} if the application should continue running, {@code false} if it should exit.
+     */
+    public boolean exitCommand() {
```

Comment:
The boolean function here does not really follow boolean naming convention. Would it be better to directly create a new instance of `ExitCommand` in the `case "exit"` block and call a method `shouldContinueLoop` belonging to `ExitCommand`?